doc/TRAINING.rst

```diff
@@ -207,9 +207,14 @@ Producing a mmap-able model is as simple as:
 Upon sucessfull run, it should report about conversion of a non-zero number of nodes. If it reports converting ``0`` nodes, something is wrong: make sure your model is a frozen one, and that you have not applied any incompatible changes (this includes ``quantize_weights``\ ).
 
 Continuing training from a release model
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+----------------------------------------
 
-If you'd like to use one of the pre-trained models released by Mozilla to bootstrap your training process (transfer learning, fine tuning), you can do so by using the ``--checkpoint_dir`` flag in ``DeepSpeech.py``. Specify the path where you downloaded the checkpoint from the release, and training will resume from the pre-trained model.
+There are two easy ways to make use of a pre-trained DeepSpeech model: fine-tuning or transfer-learning. Choosing which one to use depends on your target dataset, and it's simple decision. Does your data use the same alphabet as the release model? If you are using the same alphabet as an English release model (i.e. `a-z` plus `'`) then the output layer will match your data, and you can fine-tune the existing parameters. However, if you want to use a new alphabet (e.g. Cyrillic `а`, `б`, `д`), the output layer of DeepSpeech will *not* match your data. In this case, you should use transfer-learing (i.e. remove the output layer, and reinitialize a new output layer that matches your target character set.
```

Comment:
You should mention how this interacts with the UTF-8 case.

Reply:
How does it interact with the UTF-8 case?

Reply:
I don't know, you're the world's expert on this code :)

However, if it does or doesn't, I think you should say so either way because that's a question readers will have.

Reply:
added NB about UTF8